packages/fiori/src/ViewSettingsDialog.js

```diff
@@ -300,26 +517,56 @@ class ViewSettingsDialog extends UI5Element {
 	 * Stores current settings as confirmed and fires <code>confirm</code> event.
 	 */
 	_confirmSettings() {
-		this._confirmedSettings = this._currentSettings;
-		this.fireEvent("confirm", {
-			sortOrder: this._confirmedSettings.sortOrder && this._confirmedSettings.sortOrder.innerText,
-			sortBy: this._confirmedSettings.sortBy ? this._confirmedSettings.sortBy.innerText : "",
-		});
 		this.close();
+		this._confirmedSettings = this._currentSettings;
+		this._confirmedFilters = [...this._currentFilters];
```

Comment:
Could we have more than one selected filters in this._confirmedFilters as there could be selected FilterOptions
in more than one filter?

Reply:
Yes, we can